ludwig/benchmarking/summarize.py

```diff
@@ -85,102 +87,15 @@ def build_summary(
     """
     config = load_yaml(bench_config_path)
     downloaded_names = set(download_artifacts(config, base_experiment, experimental_experiment, download_base_path))
+    print(downloaded_names)
     experiment_diffs = []
     for n in downloaded_names:
         if isinstance(n, tuple) and len(n) == 2:
             (dataset_name, local_dir) = n
-            e = ExperimentsDiff(dataset_name, base_experiment, experimental_experiment, local_dir)
-            if not e.empty:
+            try:
+                e = build_metrics_diff(dataset_name, base_experiment, experimental_experiment, local_dir)
                 experiment_diffs.append(e)
+            except Exception:
+                print("Exception encountered while creating diff summary for", dataset_name)
+                print(traceback.format_exc())
```

Comment:
Instead of print, let's use logging.warning() or logging.info(). Or raising an error if crashing is appropriate.